pkgs/desktops/gnome-3/misc/libgda/default.nix

```diff
@@ -11,16 +17,23 @@ stdenv.mkDerivation rec {
     })
   ];
 
-  configureFlags = [
-    "--enable-gi-system-install=no"
+  NIX_CFLAGS_COMPILE = stdenv.lib.optionals mysqlSupport [
+    "-L${mysql.connector-c}/lib/mysql"
+    "-I${mysql.connector-c}/include/mysql"
   ];
 
+  configureFlags = with stdenv.lib; [ "--enable-gi-system-install=no" ]
+    ++ (optional (mysqlSupport) "--with-mysql=yes")
+    ++ (optional (postgresSupport) "--with-postgres=yes");
+
   enableParallelBuilding = true;
 
   hardeningDisable = [ "format" ];
 
   nativeBuildInputs = [ pkgconfig ];
-  buildInputs = [ intltool itstool libxml2 gtk3 openssl ];
+  buildInputs = with stdenv.lib; [ intltool itstool libxml2 gtk3 openssl ]
```

Comment:
Actually, since you are touching this, could you move `intltool itstool libxml2` to `nativeBuildInputs`?

Reply:
Moved.